src/topojson/topojson.ml

```diff
@@ -404,12 +404,18 @@ module Make (J : Intf.Json) = struct
       | Error _ -> []
 
     let base_of_json json =
-      match
-        ( J.find json [ "objects" ],
-          J.find json [ "arcs" ],
-          J.find json [ "transform" ] )
-      with
-      | Some objects, Some arcs, Some transform ->
+      match (J.find json [ "scale" ], J.find json [ "translate" ]) with
```

Comment:
Trying to find the keys `scale` and `translate` need to be performed on the `transform` object not the `json` one.